site/src/pages/SetupPage/SetupPage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useActor, useMachine } from "@xstate/react"
+import { FC, useContext, useEffect } from "react"
+import { Helmet } from "react-helmet"
+import { useNavigate } from "react-router-dom"
+import { pageTitle } from "util/page"
+import { setupMachine } from "xServices/setup/setupXService"
+import { XServiceContext } from "xServices/StateContext"
+import { SetupPageView } from "./SetupPageView"
+
+export const SetupPage: FC = () => {
+  const navigate = useNavigate()
+  const xServices = useContext(XServiceContext)
+  const [authState, authSend] = useActor(xServices.authXService)
+  const [setupState, setupSend] = useMachine(setupMachine, {
+    actions: {
+      onCreateFirstUser: ({ firstUser }) => {
+        if (!firstUser) {
+          throw new Error("First user was not defined.")
+        }
+        authSend({ type: "SIGN_IN", email: firstUser.email, password: firstUser.password })
+      },
+    },
+  })
+  const { createFirstUserFormErrors, createFirstUserErrorMessage } = setupState.context
+
+  useEffect(() => {
+    if (authState.matches("signedIn")) {
+      return navigate("/workspaces")
+    }
+  }, [authState, navigate])
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("Setup your account")}</title>
```

Comment:
```suggestion
        <title>{pageTitle("Set up your account")}</title>
```

nit: I see this spelling more commonly in titles. Not entirely sure why 🤷🏼

Reply:
I see it split more frequently when used as a verb.